test/views/declaration/AuthorisationProcedureCodeChoiceViewSpec.scala

```diff
@@ -130,5 +126,34 @@ class AuthorisationProcedureCodeChoiceViewSpec extends UnitViewSpec with Stubs w
         saveAndReturn must haveAttribute("name", SaveAndReturn.toString)
       }
     }
+
+    onJourney(STANDARD, SIMPLIFIED, SUPPLEMENTARY, OCCASIONAL) { implicit request =>
+      "display 'Back' button that links to 'Authorisations Required' page" in {
+        val backButton = view.getElementById("back-link")
+        backButton must containMessage("site.back")
+        backButton must haveHref(routes.AdditionalActorsSummaryController.displayPage())
+      }
+
+    }
+
+    onClearance(aDeclaration(withType(CLEARANCE), withEntryIntoDeclarantsRecords(YesNoAnswers.yes))) { implicit request =>
```

Comment:
Nit: Not sure, but maybe you can avoid writing `withType(CLEARANCE)` ?

Reply:
No it is needed as you can pass any Declaration instance to this method (the method was private but I've relaxed its restriction so that I can use it to pass a declaration with EIDR set).